modular_skyrat/modules/customization/modules/mob/living/carbon/human/human.dm

```diff
@@ -83,21 +83,24 @@
 	var/underwear_button = underwear_visibility & UNDERWEAR_HIDE_UNDIES ? "Show underwear" : "Hide underwear"
 	var/undershirt_button = underwear_visibility & UNDERWEAR_HIDE_SHIRT ? "Show shirt" : "Hide shirt"
 	var/socks_button = underwear_visibility & UNDERWEAR_HIDE_SOCKS ? "Show socks" : "Hide socks"
-	var/list/choice_list = list("[underwear_button]" = 1,"[undershirt_button]" = 2,"[socks_button]" = 3,"Show all" = 4, "Hide all" = 5)
+	var/bra_button = underwear_visibility & UNDERWEAR_HIDE_BRA ? "Show bra" : "Hide bra"
+	var/list/choice_list = list("[underwear_button]" = 1, "[bra_button]" = 2, "[undershirt_button]" = 3,"[socks_button]" = 4,"Show all" = 5, "Hide all" = 6)
```

Comment:
These indexes should ideally all be defines, that would be a lot less confusing than random numbers.